test-src/manageamiset_test.js

```diff
@@ -0,0 +1,77 @@
+var slugid = require('slugid');
+var assume = require('assume');
+var debug = require('debug')('test');
+var _ = require('lodash');
+var mock = require('./mock-amisets');
+var base = require('taskcluster-base');
+
+var makeAmiSet = mock.makeAmiSet;
+
+var main = require('../lib/main');
+var helper = require('./helper');
+
+describe('AMI Set api', () => {
+
+  // var id = slugid.nice();
+  var amiSetDefinition = makeAmiSet();
+  var amiSetChanged = _.clone(amiSetDefinition);
```

Comment:
It doesn't look like this AMI set is actually different from the original.  Maybe add a new AMI to it, just to be more realistic?